pkg/config/config.go

```diff
@@ -121,10 +121,14 @@ func (c Connection) BuildFromConnParams() (*dburl.URL, error) {
 	}
 	u.RawQuery = v.Encode()
 
-	return &dburl.URL{
+	url := &dburl.URL{
```

Comment:
bad variable name.

Reply:
Yep :) Just realized it as the new linter errored on it